src/rmrk2.0.0/tools/consolidator/interactions/list.ts

```diff
@@ -44,7 +44,7 @@ export const listForSaleInteraction = async (
     );
   }
 
-  if (nft.transferable === 0 || nft.transferable >= remark.block) {
+  if (nft.transferable === 0 || nft.transferable >= remark.block || ((((nft.block - nft.transferable) < remark.block) && (nft.transferable < 0)) && (listEntity.price != 0))) {
```

Comment:
perhaps this should be a function in utils

```
const isNftTransferable = (nft: NFT, remark: Remark) => {
  return nft.block - nft.transferable >= remark.block;
}
```

Reply:
why `< 0` ? the NFT is transferable if transferable value is more than 0 right? negative numbers aren't allowed?

Reply:
> why `< 0` ? the NFT is transferable if transferable value is more than 0 right? negative numbers aren't allowed?

according to specs it should be negative to say that you want the nft to be transferable for x blocks

Reply:
https://github.com/rmrk-team/rmrk-spec/blob/master/standards/rmrk2.0.0/entities/nft.md#nft-standard

Reply:
it actually needs to be differentiated as otherwise if you want the nft to be transferable from block x onwards it'd only be transferable till block x + nft.block instead